app/src/debug/java/timur/gilfanov/messenger/debug/DebugDataRepository.kt

```diff
@@ -0,0 +1,15 @@
+package timur.gilfanov.messenger.debug
+
+import kotlinx.coroutines.flow.Flow
+
+interface DebugDataRepository {
+
+    suspend fun getSavedScenario(): DataScenario?
+    suspend fun toggleAutoActivity(enabled: Boolean)
+    suspend fun toggleNotification(show: Boolean)
+    suspend fun updateSettings(transform: (DebugSettings) -> DebugSettings)
+    suspend fun clearAllData()
```

Comment:
Why is "All" specifically mentioned?
```suggestion
    suspend fun clearData()
```